### ISSUE TITLE
XMR was lost after trying to send to another wallet

### ISSUE THREAD

**Issue by xmr2206** [monero-project/monero-gui#3953]
A very strange situation has happened. I tried to send my XMR to my another wallet but got a double spend error. Then immediately tried again and got the same error. I wanted to try for the third time, but the GUI hung on the commission calculation and I moved away from the computer. When I returned the next day, I saw that the XMR had been sent but not received by the recipient. The sender address was not specified in the details of the operation, I do not see to whom the XMR were sent. The sender field is set to **unknown recipient**. And when I asked for confirmation, I got a long string that started with **SpendProofV1**ZNDb...

I tried to check the operation through https://xmrchain.net/myoutputs - but it is clear that the recipient did not receive the money (I used transaction hash and Recipient Address and private viewkey)

Also I will trigger a wallet refresh from scratch but nothing happened.

Can anyone help me?

GUI version: 0.17.3.2-unknown (Qt 5.15.3)
Embedded Monero version: 0.17.3.2-unknown
Wallet restore height: 1446091
Wallet mode: Advanced mode (Remote node)
Graphics mode: OpenGL

**Comment by selsta** [comment]
Can you post a screenshot from the "Transactions" section in the GUI and also post your transaction id?

**Comment by xmr2206** [comment]
> Can you post a screenshot from the "Transactions" section in the GUI and also post your transaction id?
> 
> Which remote node were you using?

![image](https://user-images.githubusercontent.com/108237234/175876671-9d22c7dd-f88c-43a4-b72f-a2abef98ff67.png)

Transaction Id is e474a49ba3836a4fa21501fc9d3460e199c8d54bc5bea266820a489de2a926f7

remote node: node.moneroworld.com:18089

**Comment by plowsof** [comment]
Triggering a wallet refresh will remove the recipient address / replace it with 'Unknown Recipient'. You have confirmed using xmrchain.net that you sent the funds to another address.

**Comment by selsta** [comment]
Can you use `88.198.199.23:18081` as your remote node (it's hosted by me) and then restore all wallets you have from seed with a restore height than is lower than 2652820? You don't have to restore the wallet where you sent the transaction from.

**Comment by xmr2206** [comment]
Maybe I figured out what happened. My seed phrase was compromised via wallet.mymonero.com where I had the misfortune to go some time ago. The phrase Unknown Recipient also indicates that the transaction was not made from my wallet.

Yesterday I did a test, sent XMR to my wallet and they were also transferred by someone else. Someone monitors my wallet and initiates transactions as soon as XMR arrives there. The first time it took 2 hours. Last test time - around 11 hours.
